actionRouter.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+
+const Action = require("./data/helpers/actionModel");
+
+const actionRouter = express.Router();
+
+// Get all
+
+//Get actions by action_id
+
+//Post new action to specific action by action_id
+
+//Put action by id
+
+//Delete action by ID
+
+// Get all
+
+actionRouter.get("/", async (req, res) => {
+  try {
+    const action = await Action.get();
+    res.status(200).json({ action });
+  } catch (error) {
+    res.status(500).json({ message: "Error retrieving actions" });
+  }
+});
+
+//Get actions by action_id
+
+actionRouter.get("/:id", async (req, res) => {
+  try {
+    const action = await Action.get(req.params.id);
+    res.status(200).json({ action });
+  } catch (error) {
+    res.status(500).json({ message: "Error retrieving action" });
+  }
+});
+
+//Post new action
+
+actionRouter.post("/", async (req, res) => {
+  try {
+    const { project_id, description, notes } = req.body;
+    if (!project_id) {
+      res.status(404).json({ message: "No project with this ID" });
```

Comment:
this will trigger if the user does not supply a project_id in the request, so a mo appropriate message may be "must include a valid project_id in the request" this would be a 400 as it is triggered when they are not supplying it, not after checking the db and returning with nothing found